rslib/backend.proto

```diff
@@ -276,6 +277,10 @@ service CardsService {
   rpc SetFlag(SetFlagIn) returns (OpChanges);
 }
 
+service BrowserRowsService {
+  rpc BrowserRowForCard(CardID) returns (BrowserRow);
+}
```

Comment:
It's a bit odd to have a service for a single method - I might merge this into the search service, and maybe rename it to browser.